index.bs

```diff
@@ -176,6 +166,27 @@ run these steps:
 
 </div>
 
+A <dfn export>file system path</dfn> is an implementation-defined type
+representing a potential location of a [=/file system entry=]. For example, this
+may be to a virtual path that is mapped to real location on disk or in memory,
+may correspond directly to a path on the local file system, or may not
+correspond to any file on disk at all.
+
+Each [=/file system path=] has an associated <dfn for="file system path">locate</dfn>
+algorithm, which returns a [=/file system entry=]. Exactly how a [=/file system path=]
+maps to a [=/file system entry=] left up to individual user-agent implementations.
+
+Each [=/file system path=] has an associated <dfn for="file system path" id=entry-query-access>query access</dfn> algorithm, which takes "`read`"
```

Comment:
`query access` and `request access` should map to a FileSystemHandle, and therefore a path, but I believe it's more semantically correct to have locks correspond to an entry (for example, POSIX's [`flock` takes a file descriptor](https://man7.org/linux/man-pages/man2/flock.2.html))

Reply:
That being said... for `move()`, we need to be able to lock the destination, where an entry does not yet exist. So what we're really locking is the potential location of a future entry, a.k.a. a path

Reply:
Tentatively kept `query access` and `request access` as associated with an entry for now, but we might need to change this to support requesting access to handles which do not map to an entry

Reply:
resolving this thread, since this is being discussed elsewhere